docs/event_scopes.md

```diff
@@ -0,0 +1,57 @@
+# Event scopes
+
+To understand what event scopes are, we first need to understand what events are. This description provides a summary of the BPMN 2.0 specification and combines it with how it works in Zeebe.
+
+In BPMN, **events** are flow nodes that represent that something can happen that affects the flow of the process.
+
+There are three main types of events:
+- **start events**, i.e. a start of a process
+- **end events**, i.e. the end of a path in a process
+- **intermediate events**, i.e. indicates where something can happen between the start and end of a process; intermediate events attached to an activity boundary are called boundary events
+
+and two different flavors of events:
+- **catch events**, i.e. events that catch a trigger
+- **throw events**, i.e. events that throw a trigger (sometimes referred to as a result)
+
+So events represent where in the process "something" can happen. This "something" is called a **trigger**. When a *trigger occurs*, the engine *forwards the trigger* to the catch event. The spec also refers to this as *triggering the catch event*.
+
+There are multiple ways to **forward a trigger**:
```

Comment:
:x: I feel this section is a bit weak. Is it about the triggers or forwarding of triggers.
maybe this could also be rewritten as a table with
* Trigger
* Forwarding
* Event

or something like that

Reply:
I've taken a different approach and added a description of what happens when a trigger is forwarded. I hope this clarifies the difference.